soh/soh/Enhancements/randomizer/3drando/item_list.cpp

```diff
@@ -278,6 +278,16 @@ Item& ItemTable(const uint32_t itemKey) {
     return itemTable[itemKey];
 }
 
+Item& ItemFromGIID(const int giid) {
+    uint32_t index = 0;
+    while (index < KEY_ENUM_MAX) {
```

Comment:
```suggestion
    while (index < RED_POTION) {
```
Can’t remember what the actual last item table key is but key enum max is much bigger than the item table

Reply:
If we're down with doing more of a hard fork from 3drando, then I'll want to replace their keys enum with our randomizerTypes enums anyway. Performance with this at KEY_ENUM_MAX performs fine and is clear in what it's doing until then.